controller_manager:
  ros__parameters:
    update_rate: 60  # Hz

joint_state_broadcaster:
  ros__parameters:
    type: joint_state_broadcaster/JointStateBroadcaster

forward_velocity_controller:
  ros__parameters:
    type: forward_command_controller/ForwardCommandController
    joints:
      - joint_rotation_front_right
      - joint_rotation_front_left
      - joint_rotation_rear_right
      - joint_rotation_rear_left
    interface_name: velocity
    command_interfaces:
      - velocity
    state_interfaces:
      - position
      - velocity

forward_position_controller:
  ros__parameters:
    type: forward_command_controller/ForwardCommandController
    joints:
      - joint_steering_front_left
      - joint_steering_front_right
      - joint_steering_rear_left
      - joint_steering_rear_right
    interface_name: position
    command_interfaces:
      - position
    state_interfaces:
      - position
      - velocity
